### ISSUE TITLE
Drasil Research Group Meeting, THURSDAY, June 1, 2:30pm in ITB/225(?)

### ISSUE THREAD

**Issue by smiths** [JacquesCarette/Drasil#3451]
NOTE the deviation form our usual pattern.  The meeting will be on Thursday this week, likely in ITB/225, but that hasn't been confirmed.

The agenda for this coming week is as follows:

1. Generating test cases and test documentation (@samm82)
  - https://github.com/samm82/Drasil/issues/43
2. Discuss scope of @balacij PhD thesis, using [OGS application](https://github.com/balacij/ogs-d-application/blob/main/build/ogs.pdf) as a starting point.
3. Questions/Comments/Concerns/Discussion/Issues (please add after your name)
   a. @samm82: Branch deletion: [`quantPlate`](https://github.com/JacquesCarette/Drasil/tree/quantPlate) (#3295), [`projectileDemos`](https://github.com/JacquesCarette/Drasil/tree/projectileDemos), those in my fork? 
   b. @balacij 
   c. @hrzhuang 
   d. @ManvendraJani 
   e. @harmanpreet-sagar 
4. Open Discussion/Questions

@samm82 the first agenda items is to discuss your MASc work.  I added a link to the discussion of the test plan, but I can't find the link to the discussion that you, @JacquesCarette and I had about independent module testing, assumptions, etc.  Can you add a link to the discussion, and to anything else you would like to highlight for discussion on your current research.

@hrzhuang, @harmanpreet-sagar and @ManvendraJani, please add under "Questions/Comments/Concerns/Discussion/Issues" any questions you might currently have.  Typically this involves a link to a specific issue, or issues, that you are struggling with.

Future Agenda items
- Overview of [BeamBend](https://github.com/balacij/Drasil/blob/bmBndngNoFunctions/People/Jason/CAS%20741/srs/build/srs.pdf) (@balacij)

Virtual attendees: @cd155, @tingyuw

Regrets:

**Comment by hrzhuang** [comment]
I would like to talk about where I am with regards to adding vectors to GOOL and get some feedback on the direction and design decisions. Would it be a good idea (and do we have time) for me to do a presentation (~ 10 minutes) on where I am and how did I get here (instead of telling everyone to jump straight into my code)?

**Comment by smiths** [comment]
Yes @hrzhuang the group meeting is a great time for feedback on direction and design decisions.  Yes, we should have time for a 10 minute presentation.  In case participants would like a sneak peak, it would be good to include links (in the agenda) to any code you have written.  Please edit the agenda to include the topic you will present, along with any relevant links.

**Comment by hrzhuang** [comment]
I will prepare a presentation then. On the logistics side, should I connect my computer to the projector or transfer my slides to the AV computer and present from there? If I were to use my computer, do we have a dongle that I could use to connect the projector to my USB-C port?

**Comment by smiths** [comment]
The easiest way to present is to connect to Teams and share your screen.  I think most people join Teams during the meeting, even those in-person.  If necessary, we could have a computer connected to the data projector.  It doesn't have to be your computer though.  I have a usb-c to HDMI if we need it.

**Comment by smiths** [comment]
We are now confirmed for ITB/225 on Thursday.

**Comment by tingyuw** [comment]
I will be on vacation starting tomorrow and will not be able to attend the meeting tomorrow, as well as for the next two weeks (June 13th is the convocation).

**Comment by smiths** [comment]
Enjoy your vacation @tingyuw!

**Comment by cd155** [comment]
I am unlikely to be able to attend today's meeting because of a pre-scheduled event. I will probably in outdoors at the time.

**Comment by smiths** [comment]
Okay @cd155.  Since you would be the only virtual attendee, please let me know if your plans change and if you will be attending vitually.

**Comment by cd155** [comment]
@smiths, sorry. I will be outdoors at the time, so I can't make it today

**Comment by JacquesCarette** [comment]
Paper for @hrzhuang to look at: [Multi-stage programming with functors and monads: eliminating abstraction overhead from generic code](https://www.cas.mcmaster.ca/~Carette/publications/scp_metamonads.pdf)

**Comment by smiths** [comment]
@hrzhuang have a look at [ssp interslice normal forces](https://jacquescarette.github.io/Drasil/examples/ssp/SRS/srs/SSP_SRS.html#IM:crtSlpId).
